build.sbt

```diff
@@ -1,13 +1,15 @@
+Global / onChangedBuildSource := ReloadOnSourceChanges
+
 val V = new {
   val cats            = "2.8.0"
   val catsEffect      = "3.3.14"
-  val circe           = "0.15.0-M1"
-  val http4s          = "0.23.15"
-  val munit           = "0.7.29"
-  val munitCatsEffect = "1.0.7"
+  val circe           = "0.14.3"
```

Comment:
Had to downgrade due to 0.15 not having an artifact with Native 0.4 support. To be honest I'd recommend sticking to 0.14.x anyway, as it's also what http4s-circe 0.23.x (and many other libs in the ecosystem) uses.

It's only here for tests so it shouldn't be a breaking change.